pyproject.toml

```diff
@@ -44,7 +44,7 @@ tracker = "https://github.com/scikit-image/scikit-image/issues"
 [build-system]
 build-backend = "mesonpy"
 requires = [
-    "meson-python>=0.11.0",
+    "git+https://github.com/mesonbuild/meson-python",
```

Comment:
The correct syntax is "meson-python @ git+https://github.com/mesonbuild/meson-python". I would also pin the version, in case future commits to the meson-python git repository break something:
```
meson-python @ git+https://github.com/mesonbuild/meson-python@babdf709618fbeeaf59634bc2b4331ca5f7924ce"
```